nav2_system_tests/maps/empty.yaml

```diff
@@ -0,0 +1 @@
+/home/melih/ros2_all_ws/nav2_dashing_ws/src/navigation2/nav2_bringup/launch/turtlebot3_world.yaml
```

Comment:
This path looks wrong. It shouldn't begin with /home/melih and shouldn't point to the turtlebot3_world.yaml